airflow/contrib/operators/azure_container_instances_operator.py

```diff
@@ -208,15 +200,16 @@ def _monitor_logging(self, ci_hook, resource_group, name):
         last_state = None
         last_message_logged = None
         last_line_logged = None
-        for _ in range(43200):  # roughly 12 hours
+        while True:
```

Comment:
instead of while(true), should we instead have a setting for time-out value so DAGs don't just get stuck?

Reply:
Will update with a timeout parameter